packages/blocks/.eslintrc.cjs

```diff
@@ -13,6 +13,7 @@ module.exports = {
   },
   rules: {
     // @todo(mp APP-2482): move to js-config
+    'sonarjs/no-empty-collection': 'off',
```

Comment:
Not compatible with svelte binding elements to an array (used in vectorInput)

Reply:
Oh man this rule is straight up crashing (as opposed to flagging an error) for me locally. Is that what you're seeing, too?

Reply:
uh oh - nope this was working for me - but I did encounter eslint rules (TS plugin related) crashing last week in this repo specifically

Reply:
I've disabled this rule in an overrides, so you might be able to remove this line once `main` is merged in!

Reply:
worked, restored this!